### ISSUE TITLE
Lua error when entering instance

### ISSUE THREAD

**Issue by RightfullyWarped** [widxwer/Questie#1]
Hi! I get the following error as soon as I enter an instance (using the Dungeon Finder, haven't tried w/o it), and by the count number you can tell that it happens continuously. It is worth mentioning that I'm using a modified version of Mapster and an [MPQ patch](https://forum.warmane.com/showthread.php?t=424250) that includes the missing instance maps on the 3.3.5 client. Let me know if I can help in any way, by providing a minimal way of reproducing, etc. Thanks for the great port btw!

```
Message: ...erface\AddOns\Questie-335\Modules\Map\QuestieMap.lua:203: attempt to index local 'mapInfo' (a nil value)
Time: 05/07/24 18:54:29
Count: 164
Stack: ...erface\AddOns\Questie-335\Modules\Map\QuestieMap.lua:203: in function `GetScaleValue'
...erface\AddOns\Questie-335\Modules\Map\QuestieMap.lua:307: in function `callback'
Interface\AddOns\Questie-335\Compat\Compat.lua:134: in function <Interface\AddOns\Questie-335\Compat\Compat.lua:132>

Locals: mapId = nil
scaling = 1
mapInfo = nil
(*temporary) = nil
(*temporary) = "attempt to index local 'mapInfo' (a nil value)"
HBDPins = <table> {
 SetMinimapObject = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:715
 RemoveMinimapIcon = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:689
 AddWorldMapIconMap = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:785
 activeMinimapPins = <table> {
 }
 RemoveAllWorldMapIcons = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:841
 worldmapPins = <table> {
 }
 RemoveAllMinimapIcons = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:702
 UpdateWorldMap = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:538
 worldmapProvider = <table> {
 }
 AddMinimapIconWorld = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:614
 AddMinimapIconMap = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:654
 RemoveWorldMapIcon = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:827
 AddWorldMapIconWorld = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:742
 MinimapGroup = QuestieFrameGroup {
 }
 updateFrame = <unnamed> {
 }
 IsMinimapIconOnEdge = <function> defined @Interface\AddOns\Questie-335\Compat\HBD.lua:678
 Minimap = Minimap {
 }
}
C_Map = <table> {
 GetBestMapForUnit = <function> defined @Interface\AddOns\Questie-335\Compat\Compat.lua:246
 GetAreaInfo = <function> defined @Interface\AddOns\Questie-335\Compat\Compat.lua:241
 GetWorldPosFromMapPos = <function> defined @Interface\AddOns\Questie-335\Compat\Compat.lua:253
 GetMapInfo = <function> defined @Interface\AddOns\Questie-335\Compat\Compat.lua:234
}
```

**Comment by widxwer** [comment]
Should be fixed in [76f350c](https://github.com/widxwer/Questie/commit/76f350ce668da67c2fd22646620cc1364eecedf9)

**Comment by RightfullyWarped** [comment]
Can confirm it's fixed ^^
